### ISSUE TITLE
File preview doesn't work for EPUB files

### ISSUE THREAD

**Issue by Jeremy7701** [linuxmint/nemo#2578]
```
 * Nemo version 4.6.5
 * Issue with desktop or windowed nemo
 * Distribution - LMDE4
 * Graphics hardware *and* driver used
 * 64 bit
 ```

**Issue**
File preview on EPUB files launches the previewer program - but this just displays an empty window with an 'in progress' indication.
It uses small amounts of CPU, doesn't interfere with anything, doesn't preview anything.

**Steps to reproduce**
Preview any EPUB file.

**Expected behaviour**
Display something similar to Xreader or E-book viewer

**Other information**
I don't have any non-free formats to test it on.

**Comment by icarter09** [comment]
@Jeremy7701 can you provide a video of this occurring? Want to make sure I'm following the correct steps to reproduce the issue. Thanks.

**Comment by Jeremy7701** [comment]
Nemo 4.8.4
At about 1 sec - select EPUB file to be previewed
At about 2 sec - press spacebar to see preview
At about 13 sec - press spacebar to stop preview attempt.
This book is perfectly readable by Ebook-viewer and Calibre.

https://user-images.githubusercontent.com/52585119/106397594-306d8f80-6406-11eb-8a48-3e7c59192302.mp4

**Comment by Jeremy7701** [comment]
Fixed in nemo 5.2.4
